Doc/library/reprlib.rst

```diff
@@ -71,72 +73,75 @@ string instead.
 Repr Objects
 ------------
 
-:class:`Repr` instances provide several attributes which can be used to provide
-size limits for the representations of different object types,  and methods
-which format specific object types.
+.. class:: Repr(self, *, maxlevel=6, maxtuple=6, maxlist=6, maxarray=5, \
+                 maxdict=4, maxset=6, maxfrozenset=6, maxdeque=6, maxstring=30, \
+                 maxlong=40, maxother=30)
 
+   :class:`Repr` instances provide several attributes which can be used to provide
```

Comment:
Please limit lines to 80 characters in reST files (https://devguide.python.org/documenting/#use-of-whitespace).